src/make_round_config.R

```diff
@@ -1 +1,162 @@
-print("Once implemented, script will update hub-config/tasks.config based on latest list of clades to model.")
\ No newline at end of file
+## Add a new round to the hub's task config (hub-config/tasks.json)
+## This script is part of a "round open" process run via a scheduled GitHub action and
+## should be run from the hub's src/ directory.
+
+library(hubAdmin)
+library(hubUtils)
+library(tools)
+
+get_clade_file_info <- function(hub_dir) {
```

Comment:
hub_dir should be defined somewhere.

Reply:
Yep---I took your advice and roxygenized the functions in the file.